### PR TITLE
MRG: force continue past `tax genome` classification errors

### PR DESCRIPTION
**Pull request by bluegenes**
When we were doing one or a few genome classifications, it made sense to error out completely if there was an issue. Now that we have fastmultigather and can do 10s of thousands at once, It would be nice to be able to continue past errors (logging them).

**Changed behavior:**
- If there is a failed classification, notify the error and do not write that result. Continue with classification.
- Finish classification and write output file, BUT exit with an error code if there were errors, except if --force is used.
- Remove some previously useful reporting about the classification ranks, because it's too much output for large-scale classification.

### CODE REVIEW

**Comment by bluegenes** [comment]
@sourmash-bio/devs ready for review

File: tests/test_tax.py [hunk 1]
```diff
@@ -5936,10 +6045,6 @@ def test_metagenome_LIN_lingroups(runtmp):
     print(c.last_result.err)
 
     assert c.last_result.status == 0
-    assert (
```
Comment (ctb) [inline tests/test_tax.py:5939]: I don't quite understand why this is removed as output - I think it's ok, just curious about the behavioral change.

**Review by ctb — APPROVED** [review]
nice work!

File: tests/test_tax.py (hunk 1 above)
Comment (bluegenes) [reply tests/test_tax.py:5939]: This originally printed once for every query that is being summarized .. which means the logging is ~10k lines of this. It made it very difficult to see the actual errors.

It probably would be helpful to print it just once, but then I'd need to keep track of whether or not it had been printed. Will think on it.

Comment (ctb) [reply tests/test_tax.py:5939]: no need, must have just missed where it was edited out in my quick scan of the code. all good to me!

Comment (bluegenes) [reply tests/test_tax.py:5939]: got it. I just updated the description to ...be more descriptive about this :)

Comment (ctb) [reply tests/test_tax.py:5939]: (please merge ;)

**Pull request merged**